src/table.tsx

```diff
@@ -75,83 +127,207 @@ export const Table = React.forwardRef(
         };
 
         const collapsedRowsMode = responsive === 'collapse-rows';
+        const hideHeadersInMobile = hideHeaders === true || hideHeaders === 'mobile';
+        const hideHeadersInDesktop = hideHeaders === true || hideHeaders === 'desktop';
+
+        console.log(hideHeadersInDesktop, hideHeaders);
+
+        const hasActionsColumn =
+            !!content && content.some((row) => !Array.isArray(row) && row.actions.length > 0);
```

Comment:
`!!content` is always true, you have added a default value

Reply:
good catch. fixed